spec/views/inventory_foods/create.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe 'inventory_foods/create.html.erb', type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end

```

Comment:
- [ ] Kindly implement Unit test using  RSpec and Capybara testing libraries.

Reply:
Fair Enough @aimemalaika